src/test/java/unit/AuthLoginLogout.java

```diff
@@ -84,11 +84,8 @@ void checkEnabled() throws IOException {
       // login the user, so we have their credentials
       testUser.login();
 
-      // build a SAM client with the test user's credentials
-      SamService samService = new SamService(Context.getServer(), Context.requireUser());
-
       // check that the user is enabled
-      UserStatusInfo userStatusInfo = samService.getUserInfo();
+      UserStatusInfo userStatusInfo = new SamService().getUserInfo();
```

Comment:
It might be clearer if this constructor was a static method like `SamService.fromContext()`

Reply:
Good idea. Changed to `fromContext()`.